P1/style.css

```diff
@@ -0,0 +1,176 @@
+BODY {
+    background: linear-gradient(90deg, rgb(44,44,44) 50%, rgb(50,50,50) 50%,rgb(50,50,50) 55%, rgb(40,40,40) 55%);
+    background-size: 40px 40px;
+    background-color: black;
+    padding-top: 39px;
+    font-family: "Source Sans Pro";
+    /*letter-spacing: 1px;*/
+}
+
+div {
+    padding-top: 21px;
+    padding-bottom: 21px;
+}
+
+h1{
+    color: #373435;
+    margin-top: 12px;
+    margin-bottom: 7px;
+    letter-spacing: 3px;
+}
+
+p.main-text {
+    color: #817e7f;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    font-size: 19px;
+    letter-spacing: 1px;
+    line-height: 1.2;
+}
+
+footer {
+    color: #c5c5c5;
+    text-align: center;
+    margin-bottom: 50px;
+}
+
+div.read-more {
+    padding: 0px;
+    width: 155px;
+    position: relative;
+    margin-top: -31px;
+    padding-bottom: 20px;
+    background-color: #ebebeb;
+}
+
+hr.double-line {
+    margin: 8px 30px 0px 30px;
+    padding: 4px;
+    border-top: 1px solid #c2c2c2;
+    border-bottom: 1px solid #c2c2c2;
+    border-left: none;
+}
+
+a {
+    color: #909090;
+    text-decoration: none;
+}
+
+.active {
+    color: red;
+}
+
+.bottom-border {
+    margin-right: 30px;
+    margin-left: 30px;
+    padding-bottom: 26px;
+    border-bottom: 1px solid #c2c2c2;
+}
+
+.logo {
+    position: relative;
+    width: 209px;
+    height: 52px;
+    margin: -65px auto 21px auto;
+    padding: 25px;
+    border-radius: 5px;
+    background-color: #ebebeb;
+    box-shadow: -10px 10px rgba(0, 0, 0, 0.1);
+    text-align: center;
+    font-size: 40px;
+    font-weight: bold;
+    letter-spacing: 3px;
+    z-index: 9999;
+}
+
+.background-div {
```

Comment:
use more meaningful naming. `background-div` is not a good choice. So couldn't you use it for `<section>`?